### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by o4codes**
checks if room creator is in room_members
if not adds the room_creator as an admin in the room_members of the room, in the create_room endpoint

### CODE REVIEW

File: backend/endpoints/rooms.py [hunk 1]
```diff
@@ -41,6 +41,16 @@ async def create_room(
 
     DB = DataStorage(org_id)
     room_obj = Room(**request.dict(), org_id=org_id, created_by=member_id)
+
+    if (
```
Comment (zxenonx) [inline backend/endpoints/rooms.py]: Do we need parentheses here? @o4codes

**Review by zxenonx — COMMENTED** [review]
Thanks for the work.

File: backend/endpoints/rooms.py (hunk 1 above)
Comment (o4codes) [reply backend/endpoints/rooms.py]: @zxenonx 
It's been removed now.
Thanks

**Review by zxenonx — APPROVED** [review]
LGTM.

**Pull request merged**